std/array.d

```diff
@@ -1475,6 +1512,16 @@ unittest //safety, purity, ctfe ...
     assertCTFEable!dg;
 }
 
+///
```

Comment:
Is there a way to indicate to people reading the doc of split in std.array to look the doc of std.algorithm for examples ?